### PR TITLE
Bump containers image to v5.10.1

### PR DESCRIPTION
**Pull request by QiWang19**
Signed-off-by: Qi Wang <qiwan@redhat.com>

#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
Bump containers image to v5.10.1
This is to get in (from v5.10.0) https://github.com/containers/image/pull/1099/files to introduce
new options in systemContex to log out registry mirror of the pulled image.
#### Special notes for your reviewer:

The feature is needed by https://github.com/cri-o/cri-o/pull/4438

#### Does this PR introduce a user-facing change?

```release-note
Bump containers image to v5.10.1
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

File: completions/fish/crio.fish [hunk 1]
```diff
@@ -114,7 +114,7 @@ complete -c crio -n '__fish_crio_no_subcommand' -f -l seccomp-use-default-when-e
 complete -c crio -n '__fish_crio_no_subcommand' -f -l selinux -d 'Enable selinux support (default: false)'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l separate-pull-cgroup -r -d '[EXPERIMENTAL] Pull in new cgroup (default: "")'
 complete -c crio -n '__fish_crio_no_subcommand' -l signature-policy -r -d 'Path to signature policy JSON file. (default: "", to use the system-wide default)'
-complete -c crio -n '__fish_crio_no_subcommand' -f -l storage-driver -s s -r -d 'OCI storage driver (default: "")'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l storage-driver -s s -r -d 'OCI storage driver (default: "overlay")'
```
Comment (mrunalp) [inline completions/fish/crio.fish:117]: This is probably unintentional. Could you drop this change?

Comment (QiWang19) [reply completions/fish/crio.fish:117]: test / validate-completions (pull_request) failed without these changes, do we need this change?

**Pull request merged**